contracts/Aavegotchi/facets/ItemsFacet.sol

```diff
@@ -182,41 +185,78 @@ contract ItemsFacet is Modifiers {
     ///@dev A wearable cannot be equipped in the wrong slot
     ///@param _tokenId The identifier of the aavegotchi to make changes to
     ///@param _wearablesToEquip An array containing the identifiers of the wearables to equip
-    function equipWearables(uint256 _tokenId, uint16[EQUIPPED_WEARABLE_SLOTS] calldata _wearablesToEquip)
-        external
-        onlyAavegotchiOwner(_tokenId)
-        onlyUnlocked(_tokenId)
-    {
+    function equipWearables(uint256 _tokenId, uint16[EQUIPPED_WEARABLE_SLOTS] calldata _wearablesToEquip) external {
+        uint256[EQUIPPED_WEARABLE_SLOTS] memory _depositIds;
+        _equipWearables(_tokenId, _wearablesToEquip, _depositIds);
+    }
```

Comment:
I get nervous seeing the "onlyAavegotchiOwner" and "onlyUnlocked" modifiers removed from the equipWearables function. Even though they are in the internal function below, I think it'd be better to put the modifiers back in the external function. @karacurt

Reply:
Fixed [here](https://github.com/aavegotchi/aavegotchi-contracts/pull/249)